features/step_definitions/login_steps.rb

```diff
@@ -1,14 +1,28 @@
 Given(/^the login page$/) do
-  #put your code here
-  pending
+  visit('/')
 end
 
 When(/^I log in with proper credentials$/) do
-  #put your code here
-  pending
+  fill_in('username', :with => "Admin")
+  fill_in('password', :with => "Admin")
+  click_button('Login')
 end
 
 Then(/^I should see the secret page$/) do
-  #put your code here
-  pending
+  url = URI.parse(current_url)
+  expect(url.path).to eq('/dashboard')
+  expect(page.has_content?("Your username is Admin"))
```

Comment:
This seems wrong. I don't think I've ever seen `expect()` without immediately calling `.to` on it.

Maybe `expect(page).to have_content('foobar')`?